deployment/media-insights-stack.yaml

```diff
@@ -855,13 +912,15 @@ Resources:
           Fn::GetAtt:
             - WorkflowExecutionLambdaDeadLetterQueue
             - Arn
-    Type: AWS::Lambda::Function
+      Tags:
+        - Key: "environment"
+          Value: "mie"
 
   StateMachineErrorCloudWatchEvent:
```

Comment:
Does this need to be tagged?

Reply:
It does not look like AWS::Events::Rule resources support tagging. [reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-events-rule.html)